### ISSUE TITLE
Bug: Failed Fetching PHP 7.4 packages Resource temporarily unavailable

### ISSUE THREAD

**Issue by Edwin-KeXu** [Varying-Vagrant-Vagrants/VVV#2686]
### Are you using the latest stable or develop branch version of VVV?

Yes (develop)

### Is it a new VVV, or an existing VVV that used to work?

Existing, worked but now broken

### Did you use a CustomFile?

No (default)

### Whats the problem?

When I execute the vagrant up --provision command, I get an error: Installing apt-get packages returned a failure code, cleaning up apt caches then exiting

### What is the output of `vagrant status`

```shell
Processing triggers for mime-support (3.64ubuntu1) ...
Processing triggers for libc-bin (2.31-0ubuntu9.9) ...
Processing triggers for rsyslog (8.2001.0-1ubuntu1.3) ...
Processing triggers for ufw (0.36-6ubuntu1.1) ...
Processing triggers for systemd (245.4-4ubuntu3.22) ...
Processing triggers for man-db (2.9.1-1) ...
Processing triggers for dbus (1.12.16-2ubuntu2.3) ...
Processing triggers for install-info (6.7.0.dfsg.2-5) ...
Processing triggers for php7.4-cli (1:7.4.33-7+ubuntu20.04.1+deb.sury.org+1) ...
Processing triggers for php7.4-fpm (1:7.4.33-7+ubuntu20.04.1+deb.sury.org+1) ...
Errors were encountered while processing:
 php-pear
E: Failed to fetch https://ppa.launchpadcontent.net/ondrej/php/ubuntu/pool/main/p/php7.4/php7.4-xml_7.4.33-7+ubuntu20.04.1+deb.sury.org+1_arm64.deb  Could not wait for server fd - select (11: Resource temporarily unavailable) [IP: 185.125.190.52 443]
E: Failed to fetch https://ppa.launchpadcontent.net/ondrej/php/ubuntu/pool/main/p/php7.4/php7.4-bcmath_7.4.33-7+ubuntu20.04.1+deb.sury.org+1_arm64.deb  Could not wait for server fd - select (11: Resource temporarily unavailable) [IP: 185.125.190.52 443]
E: Sub-process /usr/bin/dpkg returned an error code (1)
[0;38;5;9m * Installing apt-get packages returned a failure code, cleaning up apt caches then exiting[0m[0m
[0;38;5;9m ! Main packages check and install failed, halting provision[0m[0m
[0;38;5;9m ! The [0m[1m[0;38;5;5m'main'[21m[0;38;5;9m provisioner ran into problems, the full log is available at [0m[1m[0;38;5;5m'/var/log/provisioners/2023.08.19_16-08-09/provisioner-main.log'[21m[0;38;5;9m. It completed in [0m[1m[0;38;5;5m4684[21m[0;38;5;9m seconds.[0m[0m
```

### Which Operating System are you using?

Apple MacOS (Arm64/Apple Silicon)

### Which provider are you using?

Parallels (Apple Silicon/Arm)

**Comment by tomjn** [comment]
Can you include the entire output? 7.4 is nearing end of life soon I hope
the packages haven’t been withdrawn but I can’t tell from the output. It
may even be a temporary network connectivity thing

**Comment by Edwin-KeXu** [comment]
[provisioner-main.log](https://github.com/Varying-Vagrant-Vagrants/VVV/files/12386285/provisioner-main.log)
this

**Comment by Edwin-KeXu** [comment]
vagrant version: Vagrant 2.3.7
Apple MacOS (M1)

**Comment by tomjn** [comment]
Hmm resource temporarily unavailable, there's nothing that can be done to fix that at the VVV end of PHP 7.4 is still to be used.

TLDR: it's saying something equivalent to the Ubuntu PHP 7.4 package shop doesn't open for another hour or two, try again later

**Comment by tomjn** [comment]
You may also have luck switching your machine to a different network on a different ISP, universities for example are notorious for interfering to try and save bandwidth by caching things and messing it up

**Comment by tomjn** [comment]
Closing this as the issue reported is a temporary package availability error, it isn't something VVV has control over or can influence, and will self-resolve given time.

Fixing this properly would require a re-architecture of how Apt work, I expect this will take tens of thousands of man hours across many major Linux distributions

**Comment by tomjn** [comment]
Closing this as the issue reported is a temporary package availability error, it isn't something VVV has control over or can influence, and will self-resolve given time.

Fixing this properly would require a re-architecture of how Apt work, I expect this will take tens of thousands of man hours across many major Linux distributions
